icon.svg

```diff
@@ -0,0 +1 @@
+<svg width="176" height="176" viewBox="0 0 176 176" xmlns="http://www.w3.org/2000/svg"><title>Tradeshift.Developer</title><g fill="none" fill-rule="evenodd"><path d="M17.097 176C7.464 176 0 168.536 0 159V17C0 7.464 7.464 0 17.097 0h141.806C168.536 0 176 7.464 176 17v142c0 9.536-7.464 17-17.097 17H17.097z" fill-rule="nonzero" fill="#00ADEF"/><path d="M31.826 32h110.396v111H31.826z"/><path d="M126.407 82.963c-4.669-20.503-9.49-41.713-39.379-41.713-20.29 0-36.798 16.6-36.798 37v55.5h55.197v-13.875h4.6c5.074 0 9.2-4.149 9.2-9.25V96.75h10.492s-2.714-11.137-3.312-13.787zm-45.326 5.892l-6.505 6.54L60.777 81.52a4.638 4.638 0 0 1 0-6.54l13.8-13.875 6.504 6.54L70.533 78.25l10.548 10.605zm32.198-7.335l-13.8 13.875-6.503-6.54 10.547-10.605-10.547-10.605 6.504-6.54 13.8 13.875a4.638 4.638 0 0 1 0 6.54z" fill-rule="nonzero" fill="#FFF"/></g></svg>
```

Comment:
Please, add to the "copy-static.sh" shell script a copying of this icon to a static folder so it could be used in happo testing.
E.g.:
```shell
# Copy the test icon for a header storybook story.
cp -pvf *.svg static/
```

Reply:
👍